pilot/pkg/serviceregistry/kube/controller/ambientindex_external.go

```diff
@@ -119,6 +120,7 @@ func (a *AmbientIndexImpl) handleServiceEntry(svcEntry *apiv1alpha3.ServiceEntry
 		var wl *model.WorkloadInfo
 		if event != model.EventDelete {
 			wl = a.extractWorkloadEntrySpec(we, svcEntry.GetNamespace(), svcEntry.GetName(), svcEntry, c)
+			handleWorkloadCreationTime(oldWl, wl)
```

Comment:
why don't directly set it to SE.CreationTimestamp

Reply:
You right, this can just be set to se's creation time.